From d25596e2943c785f588d500835b3608bfa231e7a Mon Sep 17 00:00:00 2001
From: mkubiak <kontakt@mkubiak.com>
Date: Thu, 4 Apr 2024 14:25:15 +0200
Subject: [PATCH] Fixed resolution for pen and touchpad

(cherry picked from commit ad248d522bbe7bc2ca08aadff0f78e3df9e4879e)

Related: RHEL-5950
---
 hwdb.d/60-evdev.hwdb | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/hwdb.d/60-evdev.hwdb b/hwdb.d/60-evdev.hwdb
index c603f9ca2c..a4431e239e 100644
--- a/hwdb.d/60-evdev.hwdb
+++ b/hwdb.d/60-evdev.hwdb
@@ -498,9 +498,11 @@ evdev:input:b0003v256Cp0068*
  EVDEV_ABS_01=::200
 
 # HUION Kamvas 19 Pro (GT-1902)
-evdev:input:b0003v256Cp006Be0110*
- EVDEV_ABS_00=::68
- EVDEV_ABS_01=::123
+evdev:input:b0003v256Cp006B*
+ EVDEV_ABS_00=::40
+ EVDEV_ABS_01=::42
+ EVDEV_ABS_35=::40
+ EVDEV_ABS_36=::42
 
 #########################################
 # Lenovo
